README.md

```diff
@@ -101,22 +108,253 @@ serial port name.
 
 ```js
 var ourBoard = new require('openbci-sdk').OpenBCIBoard();
-ourBoard.autoFindOpenBCIBoard(function(portName,ports) {
-    if(portName) {
+ourBoard.autoFindOpenBCIBoard().then((value) => {
+    if(Array.isArray(value)) {
```

Comment:
so basically if the function returns a list, then it didn't auto-detect correct?  and what if there are no devices at all attached to the computer?  how will the function handle that case? i'm wondering if the logic should be more like if(len(value)!==1) then unable to autodetect...